src/pages/projects/introduction-to-cloudflare-workers/index.tsx

```diff
@@ -0,0 +1,342 @@
+                      <Link href={lesson.path}>
+                        <a className="py-1 flex space-x-2 items-center dark:text-gray-100 text-gray-700 hover:text-blue-600 group">
+                          {/* prettier-ignore */}
+                          <div className="flex-shrink-0"><svg className="text-gray-400 dark:text-gray-400 group-hover:text-blue-600" width={18} height={18} viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><g fill="none" ><path fillRule="evenodd" clipRule="evenodd" d="M10 18a8 8 0 1 0 0-16 8 8 0 0 0 0 16zM9.555 7.168A1 1 0 0 0 8 8v4a1 1 0 0 0 1.555.832l3-2a1 1 0 0 0 0-1.664l-3-2z" fill="currentColor"/></g></svg></div>
+                          <Markdown className="prose dark:prose-dark md:dark:prose-lg-dark md:prose-lg text-gray-700 dark:text-gray-100 mt-0 text-base md:text-lg">
+                            {lesson.title}
+                          </Markdown>
+                        </a>
+                      </Link>
+                    </div>
+                  </li>
+                )
+              })}
+            </ul>
+          </div>
+        </div>
+        <div className="flex justify-center">
+          <svg
+            width="7"
+            height="155"
+            viewBox="0 0 7 155"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <line
+              x1="3.5"
+              y1="-1.5299e-07"
+              x2="3.50001"
+              y2="155"
+              stroke="#B0B0B0"
+              strokeWidth="7"
+              strokeDasharray="10 10"
+            />
+          </svg>
+        </div>
+        <div>
+          <div className="mt-10 text-center pb-12">
+            <h1 className="text-2xl md:text-4xl font-bold pb-4">
+              Completed the Portfolio Project?
+            </h1>
+            <p className="text-lg md:text-2xl leading-6 text-gray-500">
+              Let us know what you build!
+            </p>
+
+            <a
+              className="inline-flex justify-center items-center px-6 py-4 font-semibold rounded-md bg-blue-600 text-white transition-all hover:bg-blue-700 ease-in-out duration-200 mt-12"
+              title="Share on twitter"
+              href="https://twitter.com/intent/tweet?text=I%20created%20my%20portfolio%20project%20for%20the%20Introduction%20to%20Cloudflare%20Workers%20course%20on%20@eggheadio!"
+              rel="noopener"
+            >
+              Tweet @eggheadio
+            </a>
+          </div>
+        </div>
+      </div>
+    </>
+  )
+}
+
+export enum CARD_TYPES {
+  SUMMARY = 'summary',
+  SUMMARY_LARGE_IMAGE = 'summary_large_image',
+}
+
+export type Topic = {
+  name: string
+  label: string
+  title?: string
+}
+
+type ProjectBriefProps = {
+  topic: Topic
+  className: any
+  pageData?: any
+  CTAComponent?: React.FC
+  ogImage?: string
+  verticalImage?: string
+  cardType?: CARD_TYPES
+}
+
+const ProjectBrief: React.FC<ProjectBriefProps> = ({
```

Comment:
This is the component from the default topic landing pages. Love the generated og image